Index: server/src/main/java/org/example/server/ServerApp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.example.server;\r\n\r\n\r\nimport org.example.server.network.SocketServer;\r\n\r\npublic class ServerApp {\r\n    public static void main(String[] args) {\r\n\t    SocketServer  socketServer = new SocketServer();\r\n\t\tsocketServer.run();\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/src/main/java/org/example/server/ServerApp.java b/server/src/main/java/org/example/server/ServerApp.java
--- a/server/src/main/java/org/example/server/ServerApp.java	(revision ab71a1968703c65eb85723029496e9b32c367857)
+++ b/server/src/main/java/org/example/server/ServerApp.java	(date 1778560842434)
@@ -1,6 +1,5 @@
 package org.example.server;
 
-
 import org.example.server.network.SocketServer;
 
 public class ServerApp {
Index: server/src/main/java/org/example/server/network/Broadcaster.java
===================================================================
diff --git a/server/src/main/java/org/example/server/network/Broadcaster.java b/server/src/main/java/org/example/server/network/Broadcaster.java
new file mode 100644
--- /dev/null	(date 1778560953417)
+++ b/server/src/main/java/org/example/server/network/Broadcaster.java	(date 1778560953417)
@@ -0,0 +1,4 @@
+package org.example.server.network;
+
+public class Broadcaster {
+}
